### PR TITLE
Generate empty test using /gen-test ChatOpts command

### PR DESCRIPTION
**Pull request by kevindiu**
### Description:

This PR generate test files using ChatOpts command `/gen-test`.
This PR also removes the failed test implementation due to unsupported generic test function by gotests.
These tests will be implemented in the next PR.

I also tested to regenerate unimplemented test is working in this PR, to simulate the test template changes which affect all unimplemented test.

It is performed by the following steps.

1. Use ChatOpts command `/gen-test` to generate empty test file
2. Modify 1 unimplemented test
3. Generate test file by using `/gen-test` command again to re-generate test
4. Verify the last commit wether if it includes the modified changes

### Versions:

- Go Version: 1.20.3
- Docker Version: 20.10.8
- Kubernetes Version: 1.22.0
- NGT Version: 2.0.9

### Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/main/CONTRIBUTING.md) document and completed [our CLA agreement](https://cla-assistant.io/vdaas/vald).
- [x] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?

### Special notes for your reviewer:

Please mainly review this commit:
https://github.com/vdaas/vald/pull/2005/commits/e4970bd4edd71b4a8ba07fe54cd1a1ad36845f65

All other commits are generated by ChatOpts.

### CODE REVIEW

**Comment by kevindiu** [comment]
test message

**Review by vankichi — APPROVED** [review]

**Review by datelier — APPROVED** [review]

File: .github/workflows/chatops.yml [hunk 1]
```diff
@@ -558,4 +558,4 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.DISPATCH_TOKEN }}
           PR_URL: ${{ github.event.issue.pull_request.url }}
-          USERNAME: ${{ steps.check_comments_approve.outputs.COMMENTER_USERNAME }}
+          USERNAME: ${{ steps.check_comments_approve.outputs.COMMENTER_USERNAME }}
```
Comment (kevindiu) [inline .github/workflows/chatops.yml:561]:
```suggestion
```

**Review by vankichi — APPROVED** [review]

**Review by datelier — APPROVED** [review]

**Pull request merged**